### ISSUE TITLE
Different version number and frequent task cancellation

### ISSUE THREAD

**Issue by adr-sit** [disdain13/PokeRoadie#210]
Version 9.20.1 (but DOS prompt shows 9.17.1)
Happened during throw or transfer

[16:24:47] TaskCanceledException - A task was canceled. | Causing Method: Void ThrowForNonSuccess(System.Threading.Tasks.Task) | Source: mscorlib | Data: System.Collections.ListDictionaryInternal

**Comment by BoomEX-UK** [comment]
Yes, version number is out of line with latest release (no effect on bot).

Could you post the full log where this appears?

**Comment by adr-sit** [comment]
At the very beginning of the bot connection

Initializing PokrRoadie Logger @ 9/25/2016 1:16:12 AM...
[01:16:13] PokeRoadie Version 2016.9.17.1 - Free at https://github.com/disdain13/PokeRoadie
[01:16:17] Logging in via: Google
[01:16:18] Client logged in
